tests/test_package.py

```diff
@@ -114,16 +119,28 @@ def test_package_upload(self, mock_conn, package_file: str, runner) -> None:
         )
 
     @pytest.fixture
-    def packages_directory(self):
-        path = os.path.join(os.getcwd(), ".packages")
-        os.mkdir(path)
-        yield path
+    def temp_dir(self):
```

Comment:
We have 3 fixtures that change directory already. Maybe you can reuse one of them? `execute_in_tmp_dir` looks similar

Reply:
Did the huge reformat of fixtures